pymc3/glm/families.py

```diff
@@ -36,7 +36,7 @@ def __call__(self, x):
 
 
 identity = Identity()
-logit = at.nnet.sigmoid
+logit = at.sigmoid
```

Comment:
Confused why this file is here after https://github.com/pymc-devs/pymc3/pull/4682.

Reply:
weird, I'll double check where exactly my main is at